nengo/spa/actions.py

```diff
@@ -168,7 +179,9 @@ def process(self, spa):
         sources = list(spa.get_module_outputs())
         sinks = list(spa.get_module_inputs())
 
+        sorted_kwargs = OrderedDict(sorted(self.kwargs.items()))
```

Comment:
Nit: you should be able to just do `sorted(self.kwargs.items())` and then remove the `iteritems` on line 186. The `OrderedDict` part is redundant.